data/config-schema.json

```diff
@@ -75,6 +74,11 @@
     "project": {
       "description": "Name of the project",
       "type": "string"
+    },
+    "monitorPort": {
```

Comment:
Hmm how should we persist the information of what `monitorPort` is? Since `start` and `status` and any other `double` commands will be different invocations of the script, we need to have some kind of storage to keep track these internal state. Currently everything is in the config, but maybe we should introduce some kind storage?

Reply:
Yeah we should have a file that stores the current status of the network, so in case the monitor process crashes we can read from it.

It can also contain things like monitor port so if we want to allow users to customize such things, we can.